src/polyhedral/index.js

```diff
@@ -109,6 +106,8 @@ export default function(root, face, r) {
     return rotateStream;
   };
 
+  if (r !== null) proj.angle(r * degrees);
```

Comment:
It’s fine to deprecate the *angle* argument to d3.geoPolyhedral, but I wouldn’t simultaneously change the default value of *r* from -π / 6 to 0; that change isn’t backwards-compatible, and unless I’m mistaken, this doesn’t feel like an important-enough change to justify a new major version number.

Also, be careful about `!=` versus `!==` here. You want loose equals for backwards-compatibility, too.

You can write this a little more succinctly as:

```js
return proj.angle(r == null ? -30 : r);
```